internal/ignition/ignition.go

```diff
@@ -1186,26 +1184,6 @@ func (g *installerGenerator) updateIgnitions() error {
 	return nil
 }
 
-func (g *installerGenerator) UpdateEtcHosts(serviceIPs string) error {
-	masterPath := filepath.Join(g.workDir, masterIgn)
-
-	if serviceIPs != "" {
-		err := setEtcHostsInIgnition(models.HostRoleMaster, masterPath, g.workDir, GetServiceIPHostnames(serviceIPs))
```

Comment:
Small suggestion to also remove this function since it's not used anywhere else
https://github.com/openshift/assisted-service/blob/5999b410edde9e7276dcbc3ec82edfad85d45973/internal/ignition/ignition.go#L1611-L1625